plugins/dell-dock/fu-dell-dock-status.c

```diff
@@ -155,8 +177,13 @@ static void
 fu_dell_dock_status_init(FuDellDockStatus *self)
 {
 	fu_device_add_protocol(FU_DEVICE(self), "com.dell.dock");
+	fu_device_set_name(FU_DEVICE(self), "Package level of Dell dock");
+	fu_device_set_summary(FU_DEVICE(self), "A representation of dock update status");
+	fu_device_set_version_format(FU_DEVICE(self), FWUPD_VERSION_FORMAT_QUAD);
 	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_UPDATABLE);
-	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_UNSIGNED_PAYLOAD);
+	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_SIGNED_PAYLOAD);
```

Comment:
Is this really right?  The package is being signed now?  It's different from WD19/WD22 then.  There should be special casing between docks for this flag it means.

Reply:
I've just reconfirmed the status package itself is unsigned, will fix it.